### DISCUSSION TITLE
Please help me

### DISCUSSION THREAD

**Discussion by Drobotsan** [NAIST-SE/AttractingContributors#1140] [2024 Attraction Contributions]
I need help with my project : places in the world.

Come to my fork:
https://github.com/Drobotsan/Places-in-the-world

Edit my README file.
section: List of Places in the world to visit one day

Best regards

**Comment by thapakorn604** [comment]
Hi, thanks for you contribution!

I also create my request to your topic -> https://github.com/Drobotsan/Places-in-the-world/pull/4

**Reply by Drobotsan** [reply]
thank U
